app.py

```diff
@@ -39,36 +68,26 @@
     cdk_app, f"{stack_name_prefix}-load-balancer", network_stack.vpc
 )
 
-apex_service_props = ServiceProps(
-    "schematic-apex",
-    8000,
-    200,
-    f"ghcr.io/sage-bionetworks/schematic-apex:{image_version}",
-    {
-        "API_DOCS_HOST": "schematic-api-docs",
-        "API_DOCS_PORT": "8010",
-        "API_GATEWAY_HOST": "schematic-api-gateway",
-        "API_GATEWAY_PORT": "8082",
-        "APP_HOST": "schematic-app",
-        "APP_PORT": "4200",
-        "THUMBOR_HOST": "schematic-thumbor",
-        "THUMBOR_PORT": "8889",
-        "ZIPKIN_HOST": "schematic-zipkin",
-        "ZIPKIN_PORT": "9411",
-    },
+app_service_props = ServiceProps(
+    "schematic-app",
+    443,
+    1024,
+    "ghcr.io/sage-bionetworks/schematic:v0.1.90-beta",
+    {},
+    f"{stack_name_prefix}-DockerFargateStack/{environment}/ecs",
 )
```

Comment:
This PR already has a lot going on so to make reviewing easier I was planning to address scaling parameters for on a follow on PR.  I can add it in this PR if you want though.  Anyways we don't need to set to `production` scale until we deploy to production.

Reply:
> I was planning to address scaling parameters for on a follow on PR. I can add it in this PR if you want though.

As long as it's not going to change the structure a lot then I'd be fine with this to be completed in a separate PR. To note: In parallel I am creating a branch off your changes here so I can get the ball rolling on deploying out the Opentelemetry collector. I can handle large structural changes, but try to keep them to a minimum.

https://github.com/zaro0508/schematic-infra-v2/pull/1

> Anyways we don't need to set to production scale until we deploy to production.

Should the configuration of resources be extracted to environment specific if this is the case? That way we can set `dev` to be the lower values (For now), prod/stage to the higher values, then if/when we need to we can always copy the values down to `dev`. But that way - at least, the values are set up ahead of time.

Reply:
> the pattern established for the OC assumes that deployments to each environment should be the same because we use the promotion workflow dev->stage->prod.  That workflow assumes that we want each environment to be the same so that when we promote the apps/container we get the same result.

I am on-board with this pattern.